lib/patches/bundler/bundler/ruby_version.rb

```diff
@@ -15,7 +15,7 @@ def self.system
                             when "ruby"
                               ruby_version
                             when "rbx"
-                              Truffle::Support::VERSION.dup
+                              Truffle::VERSION.dup
```

Comment:
To avoid confusion, you might want to just squash this into the commit that changed it and remove all history that this was ever changed incorrectly.

Reply:
@chrisseaton This code should not be changed.
Probably a rebase error.